### ISSUE TITLE
toolchain rebuilds itself at uninstall

### ISSUE THREAD

**Issue by alexchandel** [build2/build2#133]
I recently found an old toolchain I wanted to uninstall.  Upon running ` sh build2-install-0.14.0-a.0-stage.sh --uninstall`, it rebuilt itself.  This is completely unnecessary for an uninstall.  This also risked ending up with different files, which would litter the OS with droppings if it failed to remove them.

**Comment by boris-kolpackov** [comment]
Yes, this is a known issue. What happens is you've likely upgraded some system package (e.g., the compiler) which caused the rebuild. I agree this is both unnecessary and surprising. The solution that we plan to implement is a "manifest-based uninstall" where we save the list of installed files as part of the installation and then uninstall by simply removing everything in that list.
